librz/core/cmd_descs/cmd_type.yaml

```diff
@@ -0,0 +1,130 @@
+---
+name: cmd_type
+commands:
+  - name: tc
+    summary: List loaded types in C format
+    subcommands:
+      - name: tc
+        cname: type_list_c
+        summary: List loaded types in C format with newlines
+        args:
+          - name: type
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: tcd
+        cname: type_list_c_nl
+        summary: List loaded types in C format without newlines
+        args:
+          - name: type
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: tcc
+        summary: Manage calling convention types
+        subcommands:
+          - name: tcc
+            cname: type_cc_list
+            summary: List all calling conventions
+            modes:
+              - RZ_OUTPUT_MODE_STANDARD
+              - RZ_OUTPUT_MODE_SDB
+              - RZ_OUTPUT_MODE_RIZIN
+              - RZ_OUTPUT_MODE_JSON
+            args:
+              - name: type
+                type: RZ_CMD_ARG_TYPE_STRING
+                optional: true
+          - name: tcc-
+            cname: type_cc_del
+            summary: Remove the calling convention
+            args:
+              - name: type
+                type: RZ_CMD_ARG_TYPE_STRING
+          - name: tcc-*
+            cname: type_cc_del_all
+            summary: Remove all calling conventions
+            args: []
+  - name: td
+    cname: type_define
+    summary: Define type from C definition
+    args:
+      - name: type
+        type: RZ_CMD_ARG_TYPE_STRING
+  - name: tk
+    cname: type_kuery
+    summary: Perform SDB query on types database
+    args:
+      - name: type
+        type: RZ_CMD_ARG_TYPE_STRING
+        optional: true
+  - name: tn
+    summary: Manage noreturn function attributes and marks
+    subcommands:
+      - name: tn
+        cname: type_list_noreturn
+        summary: List all noreturn references
+        modes:
+          - RZ_OUTPUT_MODE_STANDARD
+          - RZ_OUTPUT_MODE_RIZIN
+        args:
+          - name: name
+            type: RZ_CMD_ARG_TYPE_STRING
+            optional: true
+      - name: tn-
+        cname: type_noreturn_del
+        summary: Remove the noreturn reference
+        args:
+          - name: name
+            type: RZ_CMD_ARG_TYPE_STRING
+            flags: RZ_CMD_ARG_FLAG_ARRAY
```

Comment:
actually this is a bit out of place... I don't see why `tn-` accepts a list, but most other `-` commands don't. Can't we just have one arg only even here?

Reply:
In theory, yes. In practice other parts of the Rizin rely on the `tn` and `tn-` commands and call them with multiple arguments.

Reply:
I'm ok to not do this in this PR, but IMHO we should be consistent. Either all `-` commands accept a list, or none.